lib/apparmortest.pm

```diff
@@ -72,12 +85,9 @@ sub aa_tmp_prof_prepare {
 
     if ($type == 0) {
         assert_script_run "mkdir $prof_dir_tmp";
-        assert_script_run "cp -r $prof_dir/{tunables,abstractions} $prof_dir_tmp/";
-        if (!(is_sle('<15-SP4') or is_leap('<16.0'))) {    # apparmor >= 3.0
-            assert_script_run "cp -r $prof_dir/abi $prof_dir/disable $prof_dir_tmp/";
-        }
-        if (is_sle('<15') or is_leap('<15.0')) {    # apparmor < 2.8.95
-            assert_script_run "cp -r $prof_dir/program-chunks $prof_dir/disable $prof_dir_tmp/";
+        foreach my $file ("tunables", "abstractions", "abi", "disable", "program-chunks" ) {
+            # if src_dir/file exists and dst_dir/file does not exists
+            check_copy_file_dir("$prof_dir/$file", "$prof_dir_tmp/$file");
```

Comment:
`$prof_dir/$file` is a file or a directory? If it is a file,  does `-d` work for checking that? I know `-e` is used to check if the file exists.

Reply:
Thanks, fixed.